pyproject.toml

```diff
@@ -155,6 +158,7 @@ module = [
 	"anyascii.*",
 	"tensorflow.*",
 	"torchvision.*",
+    "onnxruntime.*",
```

Comment:
not aligned

Reply:
that's a github view bug see:
![Screenshot from 2024-04-24 05-12-27](https://github.com/mindee/doctr/assets/23098070/1f6d6261-43df-4b93-8d92-44c7218b6bac)